src/smartdc/lib/ntp_config.js

```diff
@@ -1,6 +1,32 @@
 #!/usr/node/bin/node
 /* vim: syn=javascript ts=8 sts=8 sw=8 noet: */
 
+/*
+ * CDDL HEADER START
+ *
+ * The contents of this file are subject to the terms of the
+ * Common Development and Distribution License, Version 1.0 only
+ * (the "License").  You may not use this file except in compliance
+ * with the License.
+ *
+ * You can obtain a copy of the license at http://smartos.org/CDDL
+ *
+ * See the License for the specific language governing permissions
+ * and limitations under the License.
+ *
+ * When distributing Covered Code, include this CDDL HEADER in each
+ * file.
+ *
+ * If applicable, add the following below this CDDL HEADER, with the
+ * fields enclosed by brackets "[]" replaced with your own identifying
+ * information: Portions Copyright [yyyy] [name of copyright owner]
+ *
+ * CDDL HEADER END
+ *
+ * Copyright 2019 Joyent, Inc.
```

Comment:
Last nit -> Copyright block is its own block after license, typically. Also the CDDL 1.0 text is shorter, see $ILLUMOS/$SRC/prototypes/prototype.c for the example.

Reply:
Oh, ok. I just `cat`'d a file at random and did it the same way.

Reply:
Upon further review... other .js files follow this convention (from way back I'd guess).

Reply:
After looking at the rest of the repo, *most* files use the "expanded" CDDL header that I used here, and there are only a handful with separate copyright block. After discussing on Slack, we've decided that it's ok to leave this as-is.